### PR TITLE
Plane trees

### PR DESCRIPTION
**Pull request by EgbertRijke**
This is a short pull request introducing plane trees and proves that the type of plane trees is equivalent to the type of full binary trees.

There are some related old files about binary trees which I will clean up in another pull request, but not here.

### CODE REVIEW

File: src/trees/plane-trees.lagda.md [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+open import trees.w-types
+
+open import univalent-combinatorics.standard-finite-types
+```
+
+</details>
+
+## Idea
+
+A {{#concept "plane tree" Agda=plane-tree}} is a finite [tree](trees.trees.md)
```
Comment (VojtechStep) [inline src/trees/plane-trees.lagda.md:35]: Turns out plane trees have a wikidata entry - https://www.wikidata.org/wiki/Q10396021

File: src/trees/full-binary-trees.lagda.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+</details>
+
+## Idea
+
+A {{#concept "plane tree" Agda=full-binary-tree}} is a finite tree that can be
+drawn on a plane with the root at the bottom, and all branches directed upwards.
+More preciesly, a plane tree consists of a root and a family of plane trees
+indexed by a
+[standard finite type](univalent-combinatorics.standard-finite-types.md).
```
Comment (VojtechStep) [inline src/trees/full-binary-trees.lagda.md:26]: Wrong prose

File: src/trees/full-binary-trees.lagda.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+open import foundation.universe-levels
+
+open import univalent-combinatorics.standard-finite-types
+```
+
+</details>
+
+## Idea
+
+A {{#concept "plane tree" Agda=full-binary-tree}} is a finite tree that can be
```
Comment (VojtechStep) [inline src/trees/full-binary-trees.lagda.md:22]: https://www.wikidata.org/wiki/Q29791667

**Review by VojtechStep — COMMENTED** [review]
That was quick 😅

File: src/trees/full-binary-trees.lagda.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+open import univalent-combinatorics.standard-finite-types
+```
+
+</details>
+
+## Idea
+
+A
+{{#concept "full binary tree" Agda=full-binary-tree WD="full binary tree" WDID="Q29791667"}}
```
Comment (VojtechStep) [inline src/trees/full-binary-trees.lagda.md:23]:
```suggestion
{{#concept "full binary tree" Agda=full-binary-tree WD="full binary tree" WDID=Q29791667}}
```

Wikidata IDs are alphanumeric, so the parses doesn't expect quotes around them

File: src/trees/full-binary-trees.lagda.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+is a finite [directed tree](trees.directed-trees.md) that can in which every
```
Comment (VojtechStep) [inline src/trees/full-binary-trees.lagda.md:24]:
```suggestion
is a finite [directed tree](trees.directed-trees.md) in which every
```

File: src/trees/plane-trees.lagda.md [hunk 6]
```diff
@@ -0,0 +1,274 @@
...
+open import trees.w-types
+
+open import univalent-combinatorics.standard-finite-types
+```
+
+</details>
+
+## Idea
+
+A {{#concept "plane tree" Agda=plane-tree WD="ordered tree" WDID="Q10396021"}}
```
Comment (VojtechStep) [inline src/trees/plane-trees.lagda.md:35]:
```suggestion
A {{#concept "plane tree" Agda=plane-tree WD="ordered tree" WDID=Q10396021}}
```

File: src/trees/full-binary-trees.lagda.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+non-leaf node has a specified left branch and a specified right branch. More
+preciesly, a full binary tree consists of a root, a left full binary subtree and
```
Comment (VojtechStep) [inline src/trees/full-binary-trees.lagda.md:26]:
```suggestion
precisely, a full binary tree consists of a root, a left full binary subtree and
```

File: src/trees/plane-trees.lagda.md [hunk 8]
```diff
@@ -0,0 +1,274 @@
...
+is a finite [directed tree](trees.directed-trees.md) that can be drawn on a
+plane with the root at the bottom, and all branches directed upwards. More
+preciesly, a plane tree consists of a root and a family of plane trees indexed
```
Comment (VojtechStep) [inline src/trees/plane-trees.lagda.md:38]:
```suggestion
precisely, a plane tree consists of a root and a family of plane trees indexed
```

File: src/trees/plane-trees.lagda.md [hunk 9]
```diff
@@ -0,0 +1,274 @@
...
+by a [standard finite type](univalent-combinatorics.standard-finite-types.md).
+The concept of plane tree is also known as the concept of _ordered tree_.
```
Comment (VojtechStep) [inline src/trees/plane-trees.lagda.md:40]: This sentence doesn't sound right - Maybe "Plane trees are also known as ordered trees"?

**Review by VojtechStep — COMMENTED** [review]
A few typos, then it's good to merge 👌

File: src/trees/plane-trees.lagda.md (hunk 9 above)
Comment (EgbertRijke) [reply src/trees/plane-trees.lagda.md:40]: Yup, that's better

**Comment by VojtechStep** [comment]
Looks good, enabling automerge

**Pull request merged**